src/infinite-scroller.ts

```diff
@@ -380,9 +380,9 @@ export class InfiniteScroller
     const sentinelHeightCss = css`var(--infiniteScrollerSentinelDistanceFromEnd, 200rem)`;
     const rowGapSizeCss = css`var(--infiniteScrollerRowGap, 1.7rem)`;
     const colGapSizeCss = css`var(--infiniteScrollerColGap, 1.7rem)`;
-    const cellMinWidth = css`var(--infiniteScrollerCellMinWidth, 10rem)`;
+    const cellMinWidth = css`var(--infiniteScrollerCellMinWidth, 16rem)`;
```

Comment:
Taking the fallback cellMinWidth value from this reference which make it correct in inspect-element on https://archive.org/search?noscript=true&query=hello 
see attached:

Reply:
![Screenshot from 2022-05-16 12-15-03](https://user-images.githubusercontent.com/50868725/168534752-26887a98-f07a-4852-873f-6c6cf5838e32.png)

Reply:
👍